code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -14,15 +14,16 @@
 
 // climbing crates - Kachnov
 /obj/structure/closet/crate/MouseDrop_T(mob/target, mob/user)
-	if (!opened)
```

Comment:
Because I made it so you can't climb onto opened crates since you can close them on you and it doesn't have a special feature, which shouldn't work as you get toppled when it's opened on you.